src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
-import { FormattedHTMLMessage } from 'react-intl';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Is this still working good? I thought that FormattedHTMLMessage allowed use html code inside the texts. `<code>src/App.js</code>` still showing itself fine?

Reply:
No. but <code>src/App.js</code> its an example code, non final content

Reply:
No entiendo, luego de este cambio funciona mal?

Reply:
Yes we can use `FormattedHTMLMessage` when needed only, In the case we have pure text, this `FormattedMessage` should do.

Reply:
Entendido, _FormattedMessage_ es limitado, pero creo que algún tipo de formatos permite.

Igualmente, como dice Fede, no necesitamos ese contenido.

Reply:
i removed now